run/GCHP/HEMCO_Config.rc.templates/HEMCO_Config.rc.fullchem

```diff
@@ -2405,11 +2577,37 @@ Warnings:                    1
+0 HTAPv3_OCPO_SHP -                                                       -       -                  - -  -       OCPO  73     10 4
+# Use CEDSv2 ship emissions for species not in HTAPv3
+0 CEDS_MOH_SHP    $ROOT/CEDS/v2021-06/$YYYY/EOH-em-anthro_CMIP_CEDS_$YYYY.nc           EOH_shp           1750-2019/1-12/1/0 C xy kg/m2/s MOH   26/90  10 5
+0 CEDS_EOH_SHP    -                                                                    -                 -                  - -  -       EOH   26/91  10 5
+0 CEDS_ROH_SHP    -                                                                    -                 -                  - -  -       ROH   26/92  10 5
+0 CEDS_C2H6_SHP   $ROOT/CEDS/v2021-06/$YYYY/C2H6-em-anthro_CMIP_CEDS_$YYYY.nc          C2H6_shp          1750-2019/1-12/1/0 C xy kg/m2/s C2H6  26     10 5
+0 CEDS_C3H8_SHP   $ROOT/CEDS/v2021-06/$YYYY/C3H8-em-anthro_CMIP_CEDS_$YYYY.nc          C3H8_shp          1750-2019/1-12/1/0 C xy kg/m2/s C3H8  26     10 5
+0 CEDS_C4H10_SHP  $ROOT/CEDS/v2021-06/$YYYY/ALK4_butanes-em-anthro_CMIP_CEDS_$YYYY.nc  ALK4_butanes_shp  1750-2019/1-12/1/0 C xy kg/m2/s ALK4  26     10 5
+0 CEDS_C5H12_SHP  $ROOT/CEDS/v2021-06/$YYYY/ALK4_pentanes-em-anthro_CMIP_CEDS_$YYYY.nc ALK4_pentanes_shp 1750-2019/1-12/1/0 C xy kg/m2/s ALK4  26     10 5
+0 CEDS_C6H14_SHP  $ROOT/CEDS/v2021-06/$YYYY/ALK4_hexanes-em-anthro_CMIP_CEDS_$YYYY.nc  ALK4_hexanes_shp  1750-2019/1-12/1/0 C xy kg/m2/s ALK4  26     10 5
+0 CEDS_C2H4_SHP   $ROOT/CEDS/v2021-06/$YYYY/C2H4-em-anthro_CMIP_CEDS_$YYYY.nc          C2H4_shp          1750-2019/1-12/1/0 C xy kg/m2/s C2H4  26     10 5
+0 CEDS_PRPE_SHP   $ROOT/CEDS/v2021-06/$YYYY/PRPE-em-anthro_CMIP_CEDS_$YYYY.nc          PRPE_shp          1750-2019/1-12/1/0 C xy kg/m2/s PRPE  26     10 5
+0 CEDS_C2H2_SHP   $ROOT/CEDS/v2021-06/$YYYY/C2H2-em-anthro_CMIP_CEDS_$YYYY.nc          C2H2_shp          1750-2019/1-12/1/0 C xy kg/m2/s C2H2  26     10 5
+0 CEDS_BENZ_SHP   $ROOT/CEDS/v2021-06/$YYYY/BENZ-em-anthro_CMIP_CEDS_$YYYY.nc          BENZ_shp          1750-2019/1-12/1/0 C xy kg/m2/s BENZ  26     10 5
+0 CEDS_TOLU_SHP   $ROOT/CEDS/v2021-06/$YYYY/TOLU-em-anthro_CMIP_CEDS_$YYYY.nc          TOLU_shp          1750-2019/1-12/1/0 C xy kg/m2/s TOLU  26     10 5
+0 CEDS_XYLE_SHP   $ROOT/CEDS/v2021-06/$YYYY/XYLE-em-anthro_CMIP_CEDS_$YYYY.nc          XYLE_shp          1750-2019/1-12/1/0 C xy kg/m2/s XYLE  26     10 5
+0 CEDS_CH2O_SHP   $ROOT/CEDS/v2021-06/$YYYY/CH2O-em-anthro_CMIP_CEDS_$YYYY.nc          CH2O_shp          1750-2019/1-12/1/0 C xy kg/m2/s CH2O  26     10 5
+0 CEDS_ALD2_SHP   $ROOT/CEDS/v2021-06/$YYYY/ALD2-em-anthro_CMIP_CEDS_$YYYY.nc          ALD2_shp          1750-2019/1-12/1/0 C xy kg/m2/s ALD2  26     10 5
+0 CEDS_MEK_SHP    $ROOT/CEDS/v2021-06/$YYYY/MEK-em-anthro_CMIP_CEDS_$YYYY.nc           MEK_shp           1750-2019/1-12/1/0 C xy kg/m2/s MEK   26     10 5
+0 CEDS_HCOOH_SHP  $ROOT/CEDS/v2021-06/$YYYY/HCOOH-em-anthro_CMIP_CEDS_$YYYY.nc         HCOOH_shp         1750-2019/1-12/1/0 C xy kg/m2/s HCOOH 26     10 5
```

Comment:
Hierarchy should be 4 here. See comment in GCClassic version of this file.

Reply:
Now fixed in 3f69125
https://github.com/geoschem/geos-chem/blob/3f69125dbdf6ae180be14d943941683f4c7b9185/run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.fullchem#L2592-L2610